downstream/assemblies/eda/assembly-eda-credential-types.adoc

```diff
@@ -0,0 +1,15 @@
+[id="eda-credential-types"]
+
+= Credential types
+
+{EDAcontroller} comes with several built-in credental types that you can use for syncing projects, running rulebook activations, executing job templates through {MenuTopAE} ({ControllerName}), fetching images from container registries, and processing data through event streams. 
+
+These built-in credential types are not editable. So if you want credential types that support authentication with other systems, you can create your own credential types that can be used in your source plugins. Each credential type contains an input configuration and an injector configuration that can be passed to an Ansible rulebook to configure your sources.
```

Comment:
I would try not to begin a sentence with "so" -- it reads as somewhat informal. I would either replace that period with a comma, or just remove the "so."